components/modal.css

```diff
@@ -0,0 +1,21 @@
+/* Modal container  box*/
+.modal-container {
+    display: none;
+    position: fixed;
+    z-index: 1;
+    padding-top: 10rem;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    overflow: auto;
+    background-color: rgba(0, 0, 0, 0.4);
+    
+  }
+  
+  /* Modal box */
+  .modal-content {
+    margin: auto;
```

Comment:
give margin: 0 auto; if you want to center it

Reply:
Thanks for suggestion , but I prefer its position there only, and actually content is inside modal container so by changing padding-top  of container we can center it.